README.md

```diff
@@ -5,9 +5,7 @@ factory_bot is a fixtures replacement with a straightforward definition syntax,
 If you want to use factory_bot with Rails, see
 [factory_bot_rails](https://github.com/thoughtbot/factory_bot_rails).
 
-**A historical note:** factory_bot used to be named factory_girl. An explanation of the name change can be found in the [old factory_girl repository](https://github.com/thoughtbot/factory_girl).
-
-_[Interested in the project name?](NAME.md)._
+**A historical note:** factory_bot [used to be named factory_girl](NAME.md). Read [this comment thread](https://github.com/thoughtbot/factory_bot/issues/921#issuecomment-231735985) for some of the reasoning behind the name change.
```

Comment:
Maybe we can remove NAME.md entirely? #921 might be sufficient explanation

Reply:
I'm probably overthinking this, but in case there are some stray links to NAME.md I think it's best to keep it separate. Also I like that the README itself is neutral on the controversy; if any further explanation or discussion is required it can go in NAME.md and keep README about the current project. So I'll just merge it as is.

Reply:
oh duh I can't merge it myself, silly me